File: java/CFLOW006.java
import java.util.Scanner;

public class CFLOW006 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int z = sc.nextInt();
        int a,b;
        int count = 0;
        while (z > 0) {
            a = sc.nextInt();
            while (a > 0) {
                b = a % 10;
                count = count + b;
                a = a / 10;
            }
            System.out.println(count);
            count=0;
            z--;
        }
    }
}

File: c++/CFLOW006.cc
#include <iostream>
using namespace std;

int main()
{
    int count = 0;
    int a, b, c;
    cin >> a;
    while (a > 0)
    {
        cin >> b;
        while (b > 0)
        {
            c = b % 10;
            count = count + c;
            b = b / 10;
        }
        cout << count << endl;
        count = 0;
        a = a - 1;
    }
}